lib/doc_builder_static_data.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# module for doc builder static data
+module DocBuilderStaticData
```

Comment:
Do we really need a new class for this?
I think the purpose is exactly the same as spec/test_data.rb?

And I don't like StaticData name in general :)))